test/e2e/singlecluster/pod_test.go

```diff
@@ -273,6 +275,16 @@ var _ = ginkgo.Describe("Pod groups", func() {
 		})
 
 		ginkgo.It("should allow to preempt the lower priority group", func() {
+			podList := corev1.PodList{}
+			podWatcher, err := k8sClient.Watch(ctx, &podList, &client.ListOptions{
```

Comment:
The watcher should be for Events, not for Pods.

Reply:
I see, let me try that.

Reply:
Done, PTAL